sqs/src/main/scala/akka/stream/alpakka/sqs/SqsModel.scala

```diff
@@ -125,89 +124,213 @@ object MessageAction {
 }
 
 /**
- * Additional identifiers for FIFO messsages
+ * Result contained in a Sqs Response.
  *
- * @see [https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-additional-fifo-queue-identifiers.html]
  */
-final class FifoMessageIdentifiers @InternalApi private[sqs] (
-    val sequenceNumber: String,
-    val messageGroupId: String,
-    val messageDeduplicationId: Option[String]
-) {
+sealed abstract class SqsResult {
 
-  /** Java API */
-  def getSequenceNumber: String = sequenceNumber
+  type Result
+
+  /**
+   * The SQS response metadata (AWS request ID, ...)
+   */
+  def responseMetadata: SqsResponseMetadata
+
+  def result: Result
 
   /** Java API */
-  def getMessageGroupId: String = messageGroupId
+  def geResponseMetadata: SqsResponseMetadata = responseMetadata
 
   /** Java API */
-  def getMessageDeduplicationId: java.util.Optional[String] = messageDeduplicationId.asJava
+  def getResult: Result = result
+}
+
+/**
+ * Messages returned by a SqsPublishFlow
+ */
+final class PublishResult @InternalApi private[sqs] (
```

Comment:
No `Sqs` prefix? Was that intentional?

Reply:
The `MessageActions` are not prefixed with `Sqs`.

But you're right, I'm inconsistent with the naming: the super class are prefixed but not the implementations. I should either remove or prepend it everywhere.